### PR TITLE
Definition of Layout and Routing

### PR DESCRIPTION
**Pull request by dilane3**
What is done:

- Routing ✅
- Dashboard Layout (only for desktop) ✅
- Input Component ✅
- Sidebar Button Component ✅
- Card Container, to show elements ✅
- Typography Component ✅
- Divider Component ✅

### CODE REVIEW

File: templates/dashboard/model1/src/app/root/pages/ui/atoms.page.tsx
Comment (konnofuente) [inline templates/dashboard/model1/src/app/root/pages/ui/atoms.page.tsx:1]: Is better if we avoid using svg code directly in our cde base

**Review by konnofuente — CHANGES_REQUESTED** [review]
You did great all i can request is to 
-avoid using hardcoded svg that will be all n my side 

thanks

File: templates/dashboard/model1/src/app/root/pages/ui/atoms.page.tsx
Comment (dilane3) [reply templates/dashboard/model1/src/app/root/pages/ui/atoms.page.tsx:1]: Resolved

**Comment by jeansamist** [comment]
I think it's not integrated exactly as on the model. I will create good consistency between the model and the theme

**Review by konnofuente — APPROVED** [review]

**Pull request merged**